### PR TITLE
fix: phpstan issues

### LINKED ISSUE #287: Drupal 10, PHP 8.2 and PHPStan issues
The work on https://github.com/localgovdrupal/localgov_project/pull/131 has highlighted some PHP 8.2 and PHPStan issues which need fixing.

```
 ------ --------------------------------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/localgov_alert_banner/src/Form/AlertBannerEntityForm.php                                                   
 ------ --------------------------------------------------------------------------------------------------------------------------- 
  223    Method Drupal\localgov_alert_banner\Form\AlertBannerEntityForm::save() should return int but return statement is missing.  
 ------ --------------------------------------------------------------------------------------------------------------------------- 

 ------ ---------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/localgov_alert_banner/src/Form/AlertBannerEntityRevisionRevertTranslationForm.php                                             
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------- 
  87     Access to an undefined property Drupal\localgov_alert_banner\Form\AlertBannerEntityRevisionRevertTranslationForm::$AlertBannerEntityStorage.  
         💡 Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property                                                          
 ------ ---------------------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------------------------- 
  Line   modules/contrib/localgov_alert_banner/src/Form/AlertBannerEntityTypeForm.php                                                   
 ------ ------------------------------------------------------------------------------------------------------------------------------- 
  62     Method Drupal\localgov_alert_banner\Form\AlertBannerEntityTypeForm::save() should return int but return statement is missing.  
 ------ ------------------------------------------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------- 
  Line   modules/contrib/localgov_alert_banner/tests/src/Functional/AlertBannerBlockTest.php        
 ------ ------------------------------------------------------------------------------------------- 
  51     Method Drupal\Tests\BrowserTestBase::drupalLogout() invoked with 1 parameter, 0 required.  
 ------ ------------------------------------------------------------------------------------------- 

 ------ ------------------------------------------------------------------------------------------------------------ 
  Line   modules/contrib/localgov_alert_banner/tests/src/Functional/PermissionsTest.php                              
 ------ ------------------------------------------------------------------------------------------------------------ 
  71     Access to an undefined property Drupal\Tests\localgov_alert_banner\Functional\PermissionsTest::$adminUser.  
         💡 Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property                        
  74     Method Drupal\Tests\BrowserTestBase::drupalLogout() invoked with 1 parameter, 0 required.                   
 ------ ------------------------------------------------------------------------------------------------------------ 

 ------ -------------------------------------------------------------------------------------- 
  Line   modules/contrib/localgov_alert_banner/tests/src/Kernel/AlertBannerBlockOrderTest.php  
 ------ -------------------------------------------------------------------------------------- 
  100    Variable $result might not be defined.                                                
  134    Variable $result_2 might not be defined.                                              
  159    Variable $result_3 might not be defined.                                              
  215    Variable $result might not be defined.                                                
 ------ -------------------------------------------------------------------------------------- 
```

### PR DESCRIPTION
**Pull request by millnut**
Fix issues reported in https://github.com/localgovdrupal/localgov_alert_banner/issues/287

### CODE REVIEW

File: src/Form/AlertBannerEntityRevisionRevertTranslationForm.php [hunk 1]
```diff
@@ -84,7 +84,7 @@ protected function prepareRevertedRevision(AlertBannerEntityInterface $revision,
     $revert_untranslated_fields = $form_state->getValue('revert_untranslated_fields');
 
     /** @var \Drupal\localgov_alert_banner\Entity\AlertBannerEntityInterface $default_revision */
-    $latest_revision = $this->AlertBannerEntityStorage->load($revision->id());
+    $latest_revision = $this->alertBannerEntityStorage->load($revision->id());
```
Comment (millnut) [inline src/Form/AlertBannerEntityRevisionRevertTranslationForm.php:87]: Casing fix as AlertBannerEntityRevisionRevertForm has this as `alertBannerEntityStorage` not `AlertBannerEntityStorage`

**Review by finnlewis — APPROVED** [review]
Tests pass and looks good to me, thank @millnut !

Happy to have second opinion from @andybroomfield if needed.

**Review by andybroomfield — APPROVED** [review]
All seems sensible, and tested the functionality of the banner still works.

**Comment by andybroomfield** [comment]
As an aside, the alert banner entity was genereted with an older version of the Drush entity generate plugin, and before that it used ECK, so we might want to look at regenerting that on the modern version later.

**Pull request merged**